Data Prepper ToC Update (#2514)
opensearch-project/documentation-website: _data-prepper/getting-started.md

    opensearchproject/data-prepper:latest
```

> This sample pipeline configuration above demonstrates a simple pipeline with a source (`random`) sending data to a sink (`stdout`). For more examples and details about more advanced pipeline configurations, see [Pipelines]({{site.url}}{{site.baseurl}}/clients/data-prepper/pipelines/).

Reviewer: ```suggestion
The preceding sample pipeline configuration demonstrates a simple pipeline with a source (`random`) sending data to a sink (`stdout`). For further detailed examples of more advanced pipeline configurations, see [Pipelines]({{site.url}}{{site.baseurl}}/clients/data-prepper/pipelines/).
```

Reviewer 2: +1 @kolchfa-aws
